blog.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta charset="utf-8">
+  <title>Adopt-A-Dog</title>
+  <link href="https://fonts.googleapis.com/css?family=Milonga%7COpen+Sans&display=swap" rel="stylesheet">
+  <link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+
+<body>
+  <header class="hero-image">
+    <div>
+      <img src="images/adoptadog-logo.svg" alt="Adopt A Dog logo">
+      <h1>Seattle's Premiere Dog Adoption Agency</h1>
+    </div>
+    <div class="cart white">
+      <img class="small" src="images/cart-icon.svg" alt="shopping cart icon">  $123.45
+      <button class="checkout" formaction="checkout.html">Checkout</button>
+    </div>
+  </header>
+
+  <nav>
+    <ul>
+      <li>
+        <a href="index.html" class="done dark">Home</a>
+        <a href="dogs.html" class="dark">  Dogs</a>
+        <a href="blog.html" class="dark">  Blog</a>
+      </li>
+    </ul>
+  </nav>
+
+  <section class="clear margin-top">
+    <h2>Adoptadog Blog</h2>
+
+    <img class="blog" src="images/blog-1.jpg" alt="person hugging dog">
+    <h3>Traveling with Your Dog</h3>
+    <p class="intro">Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor
+          incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud
+          exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute
+          irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla
+          pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia
+          deserunt mollit anim id est laborum.</p>
+  </section>
+
+  <section class="clear margin-top">
```

Comment:
I'd look for more semantic names for these classes than `margin-top` . Maybe because it's a wrapper around blogs it could be `dog-container` or something?